src/components/message-box/message-box.module.css

```diff
@@ -6,6 +6,10 @@
 .borderRadius {
   border-radius: token('border.radius.regular');
 }
+.subtle {
+  background-color: token('color.background.neutral.0.background') !important;
```

Comment:
Necessary to use `!important` here?

Reply:
Nope, that was from before I added `backgroundColor: subtle ? undefined : ...` 🙃